static/app/constants/releases.tsx

```diff
@@ -0,0 +1,11 @@
+export enum ReleasesSortOption {
+  CRASH_FREE_USERS = 'crash_free_users',
+  CRASH_FREE_SESSIONS = 'crash_free_sessions',
+  USERS_24_HOURS = 'users_24h',
+  SESSIONS_24_HOURS = 'sessions_24h',
+  SESSIONS = 'sessions',
+  DATE = 'date',
+  BUILD = 'build',
+  SEMVER = 'semver',
+  ADOPTION = 'adoption',
```

Comment:
@philipphofmann can you confirm that we would like to provide all these sort by options?